packages/client/package.json

```diff
@@ -13,7 +13,8 @@
   "dependencies": {
     "axios": "^0.21.1",
     "jose": "^3.14.3",
-    "openid-client": "^4.7.4"
+    "openid-client": "^4.7.4",
```

Comment:
nit: this dependency can be removed? as well as `axios`

Reply:
ignore me about `axios`. i'm thinking about changing `core` to use `axios` for cross-repo consistency.